### ISSUE TITLE
Readonly mode in embedded script

### ISSUE THREAD

**Issue by dderevjanik** [jsitor/jsitor#34]
Hi,

do you have any plans to make embedded script editable ? It would be really helpful for making documentation with live examples.

**Comment by ashvin777** [comment]
Hi @dderevjanik , 

Do you mean to allow editing in the embeddable view (https://jsitor.com/embed/Y7MXyI_la) which is currently in read-only mode?

Also, will that be a temporary change or you would like to save the content from the embeddable view itself?

**Comment by dderevjanik** [comment]
Yes, being able to edit embedded content directly without a visiting jsitor. 
It would be useful mostly for library documentation, where you want to let user to change something in your example and instantly see result.

**Comment by ashvin777** [comment]
Ok. We can allow temporary changes which will not be allowed to save in the original snippet contents. It will be used for testing only.

**Comment by dderevjanik** [comment]
Thanks, that would be really helpful :)

**Comment by yairEO** [comment]
I also want editable embeds which cannot be saves. 
It was mentioned here that it would be done. last year... 
any news?
